spec/javascript/app/webauthn_spec.js

```diff
@@ -153,7 +155,10 @@ describe('WebAuthn', () => {
 
   describe('verifyWebauthnDevice', () => {
     const userChallenge = '[1, 2, 3, 4, 5, 6, 7, 8]';
-    const credentialIds = 'Y3JlZGVudGlhbDEyMw==,Y3JlZGVudGlhbDQ1Ng=='; // Base64-encoded 'credential123,credential456'
+    const credentials = [
+      { id: 'Y3JlZGVudGlhbDEyMw==', transports: ['usb'] }, // base64-encoded 'credential123'
+      { id: 'Y3JlZGVudGlhbDQ1Ng==', transports: ['internal', 'hybrid'] }, // base64-encoded 'credential456'
```

Comment:
I see we have this comment style elsewhere in the file, but would it be cleare/terser to do:
```suggestion
      { id: btoa('credential123'), transports: ['usb'] },
      { id: btoa('credential456'), transports: ['internal', 'hybrid'] },
```

Reply:
Yeah, I like that a lot better 👍